docs/product-assembly_pam-on-openshift/src/main/asciidoc/environment-immutable-server-proc.adoc

```diff
@@ -0,0 +1,50 @@
+[id='environment-immutable-server-proc']
+= Deploying an immutable {KIE_SERVER}
+
+To deploy an immutable {KIE_SERVER}, use the `rhpam70-prod-immutable-kieserver.yaml` template file. You can download this file from `{URL_BASE_GITHUB_PAM}/templates`.
+
+This procedure also retrieves the source code for any processes that must run on this server and builds the processes (KJAR files). 
+
+You can configure the {KIE_SERVER} to connect to Smart Router and to {CENTRAL} Monitoring. If you use the server with {CENTRAL} Monitoring, you must ensure that the same versions of KJAR files are uploaded to the Maven repository that the {CENTRAL} Monitoring instance ises.
+
+.Procedure
+
+. Use one of the following methods to deploy the template:
+.. In the OpenShift Web UI, select *Add to Project >  Import YAML / JSON* and then select or paste the `rhpam70-prod-immutable-kieserver.yaml` file. In the *Add Template* window, ensure *Process the template* is selected and click *Continue*.
+.. To use the OpenShift command line console, prepare the following command line:
++
+[subs="verbatim,macros"]
+----
+oc process -n <project-name> -f <template-path>/rhpam70-prod-immutable-kieserver.yaml -v ATTRIBUTE=value -V ATTRIBUTE=value... | oc create -n <project-name> -f -
+----
++
+In this command line:
++
+* Replace `<project-name>` with the name of your project
+* Replace `<template-path>` with the path to the downloaded template file.
+* Use as many `ATTRIBUTE=value` pairs as needed to set the required attributes. You can view the template file to see descriptions for the attributes.
++
+. Set the following mandatory attributes:
+** *Application Name* (`APPLICATION_NAME`): the name of the OpenShift application. It is used in the default URL for {KIE_SERVER}.
```

Comment:
It is (also and more importantly) used to create a distinct set of deployment configs, services, routes, labels and artifacts and distinguish different applications built using the same or similar templates within the same project.